client/src/utils/QueryFieldsUtil.ts

```diff
@@ -95,6 +95,12 @@ const QUERY_FIELDS_MAP: QueryFieldMap = {
     baseType: 'integer',
     lexType: 'integer',
   },
+  DOCS_NODE_TITLE: {
+    ..._field('docsTitle', 'Document Title'),
```

Comment:
We are really searching for a keyword included in the doc title, right? Wouldn't this be just called `Keyword` for keyword search?